haystack/nodes/reader/farm.py

```diff
@@ -407,6 +410,7 @@ def distil_prediction_layer_from(
         distillation_loss_weight: float = 0.5,
         distillation_loss: Union[str, Callable[[torch.Tensor, torch.Tensor], torch.Tensor]] = "kl_div",
         temperature: float = 1.0,
+        processor: Optional[Processor] = None,
```

Comment:
At the time, I thought that it would make sense to be able to specify this in other cases as well. But you are right, it is probably just confusing as most processors would not work anyways.

Reply:
Okay, understood. In that case I'd say we remove the processor parameter here.

Reply:
Yes, I agree. I have already removed the parameter.

Reply:
If I am not mistaken these changes have not been pushed yet. 🤔

Reply:
Oh, my mistake. Now they should be pushed.